web/src/flow/stages/authenticator_validate/AuthenticatorValidateStageCode.ts

```diff
@@ -48,6 +50,8 @@ export class AuthenticatorValidateStageWebCode extends BaseDeviceStage<
 
     deviceIcon(): string {
         switch (this.deviceChallenge?.deviceClass) {
+            case DeviceClassesEnum.Email:
+                return "fa-key";
```

Comment:
Icon to be refined.

Btw, the icons of SMS and TOTP below are not consistent with the other ones defined